### DISCUSSION TITLE
How can we apply force/torque to specific position like pybullet's applyExternalForce/Torque?

### DISCUSSION THREAD

**Discussion by keiohta** [haosulab/SAPIEN#56] [Q&A] [answered]
Hi, I would like to apply an external force to a specific point to control an object. Can SAPIEN have such API? I think such a function would be very useful for developers who want to manipulate objects without explicitly preparing a robotic hand. I imagine pybullet's `applyExternalForce/Torque`-like method (from [this guide](https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit#heading=h.mq73m9o2gcpy)) would be great.

**Comment by fbxiang** [comment, marked as answer]
We have `actor.add_force_at_point` and `actor.add_force_torque` provided.
https://github.com/haosulab/SAPIEN/blob/master/python/pysapien_content.hpp#L943-L959

**Reply by keiohta** [reply]
Thanks for the quick reply! I confirmed force can be applied by using `actor.add_force_at_point`.
